src/inventory_ui.h

```diff
@@ -55,7 +56,8 @@ class inventory_entry
             custom_invlet( entry.custom_invlet ),
             stack_size( entry.stack_size ),
             custom_category( entry.custom_category ),
-            enabled( entry.enabled ) {}
+            enabled( entry.enabled ),
+            cached_name( entry.cached_name ) {}
```

Comment:
Very minor issue, GCC throws a warning if a constructor initializes class members in an order other than the order in which they are declared, including in initializer lists.
The fix is to add the cached_name initializer between the initializers for custom_invlet and stack_size.

Reply:
I will make new PR. Thx!